workloads/maas-gateway-perf/run.sh

```
@@ -0,0 +1,552 @@
+    git clone --branch "${MAAS_REF}" --depth 1 \
+      https://github.com/opendatahub-io/models-as-a-service.git "${maas_dir}"
+  fi
+
+  if [[ "${OPERATOR_TYPE}" == "rhoai" ]]; then
+    INFRA_NS="redhat-ai-gateway-infra"
+  else
+    INFRA_NS="odh-ai-gateway-infra"
+  fi
+
+  (
+    for i in $(seq 1 120); do
```

Shellcheck (0.11.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)